modules/core/shared/src/main/scala/lucuma/core/model/ids.scala

```diff
@@ -8,7 +8,6 @@ import eu.timepit.refined.auto._
 object Asterism       extends WithId('a')
 object Atom           extends WithId('m')
 object Configuration  extends WithId('x')
-object ConstraintSet  extends WithId('c')
```

Comment:
I guess we could give `c` to Configuration if we wanted.

Reply:
Bonus!